packages/pluggableWidgets/accordion-web/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
 - We renamed the advanced options property to emphasize this property is about the advanced properties' activation.
 - We made some small code improvement.
+- We fixed an issue that breaks accessibility.
```

Comment:
Can we be more precise in the fix description? Feels to me that we are basically saying the a11y never worked, which is not true.

Reply:
Agreed, the main problem was that we had non unique identifiers that caused duplicate id attributes, which can cause inconsistencies. Furthermore, it's only in certain scenarios that it will happen (when you have more than 1 widget instance) and then still it's situational rather than guaranteed. 

Maybe something along the lines of "We fixed an issue that widgets could be assigned duplicate identifiers which caused inconsistencies in accessibility under certain circumstances". You would also have to make a new section "### Fixed"

Reply:
I agree. I changed the log description.